cassandra-service.yaml

```diff
@@ -11,3 +11,24 @@ spec:
     - port: 9042
   selector:
     app: cassandra
+---
+apiVersion: batch/v1
+kind: Job
+metadata: {name: scalable-cassandra-deployment-on-kubernetes-metrics}
+spec:
+  template:
+    metadata: {name: scalable-cassandra-deployment-on-kubernetes-metrics}
+    spec:
+      containers:
+        - env:
+            - {name: config, value: '{
+                "repository_id": "Scalable-Cassandra-deployment-on-Kubernetes",
+                "target_runtimes": ["Kubernetes Cluster"],
```

Comment:
target_services is also a required field. Even though Cassandra doesn't use any service, we still need to keep the target_services field and leave it with blank array so it would generate something like `"target_services": [null],`